### PR TITLE
feat(web) skilvul course

### PR DESCRIPTION
**Pull request by ibnuimam**
- [ADD] skilvul api get token
- [ADD] skilvul api get products (courses)
- [MODIFY] implement api into course wrapper home page
- [MODIFY] implement api into course page

### CODE REVIEW

File: svgs/no-image.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="621.127px" height="621.127px" viewBox="89.437 -10.563 621.127 621.127"
```
Comment (azhariemuhammad) [inline svgs/no-image.svg:2]: Ini lupa diminify bang

**Review by azhariemuhammad — APPROVED** [review]
Approved with suggestion

File: pages/course.js [hunk 2]
```diff
@@ -1,49 +1,23 @@
...
-      },
-    ];
-    setCourses(dataDummy);
-  };
-
-  useEffect(() => {
-    getCourses();
-  }, []);
+  let courses;
+  const { data: courseRes } = useSWR(
```
Comment (azhariemuhammad) [inline pages/course.js:14]: oh sama ini bang. belum handle loading state.
ntr tambahin isLoading trus render shimmer card  aja.
bisa pake component ini 
`components/Shimmer/Card.js`

File: svgs/no-image.svg (hunk 1 above)
Comment (ibnuimam) [reply svgs/no-image.svg:2]: ok siph.

File: pages/course.js (hunk 2 above)
Comment (ibnuimam) [reply pages/course.js:14]: o udah ada, ok siph.

Comment (ibnuimam) [reply pages/course.js]: o iya klo paje swr gak bis setState di dalam kondisinya si swr ya, jadi buat loading pake isValidate dari si swrnya.

Comment (azhariemuhammad) [reply pages/course.js]: simplenya untuk handle loading bisa gini aja bang @ibnuimam 
`{ data, error } = useSWR(key, fetcher);
return { loading: !data.length && !error }
`

klo isValidate kayaknya beda case deh.

**Pull request merged**